packages/teleport/src/services/session/types.ts

```diff
@@ -0,0 +1,15 @@
+export type RenewSessionRequest = {
+  requestId?: string;
+  switchback?: boolean;
```

Comment:
switchback -> `onlyStaticRoles` ?

Reply:
I prefer to stick with `switchback` b/c its shorter/simpler and i don't foresee `switchback` term being used for other meanings